sui/its.js

```diff
@@ -19,48 +19,43 @@ function parseTrustedChains(config, trustedChain) {
 }
 
 async function setupTrustedAddress(keypair, client, config, contracts, args, options) {
-    const [trustedChain, trustedAddress] = args;
+    const [trustedChain] = args;
 
-    const { ITS: itsConfig } = contracts;
+    const { InterchainTokenService: itsConfig } = contracts;
 
-    const { OwnerCap, ITS } = itsConfig.objects;
+    const { OwnerCap, InterchainTokenService } = itsConfig.objects;
 
     const txBuilder = new TxBuilder(client);
 
     const trustedChains = parseTrustedChains(config, trustedChain);
```

Comment:
This method is a bit weird. It has some specialized logic for all EVM chains, but doesn't really make sense. We have other non EVM chain ITS contracts as well that should be included

Reply:
If `all` is provided, we should just include the full list of chain's axelarIds where `chain.contracts?.InterchainTokenService.address` is set. Makes the initial setup easy after deployment

Reply:
out of scope for this, maybe ask euro to make that change since he wrote this logic in the first place

Reply:
@Foivos Can you coordinate with him to have this updated?

Reply:
i think done, please check